app/views/selfassessment/doYouKnowOtherCredentials.scala.html

```diff
@@ -0,0 +1,64 @@
+@*
+ * Copyright 2019 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *@
+
+@import controllers.auth.requests.UserRequest
+@import config.ConfigDecorator
+@import util.LocalPartialRetriever
+@import uk.gov.hmrc.renderer.TemplateRenderer
+@import tags._
+@import uk.gov.hmrc.play.views.html.helpers._
+
+@(wrongCredentialsUserForm: Form[dto.SAWrongCredentialsDto])(implicit request: UserRequest[_], configDecorator: ConfigDecorator, partialRetriever: LocalPartialRetriever, templateRenderer: TemplateRenderer, messages: play.api.i18n.Messages)
+
+@main(messages("title.do_you_know_other_credentials.h1"),
+    Some(messages("label.your_personal_tax_account")),
+    backlinkUrl = Some(controllers.routes.SaWrongCredentialsController.landingPage().url)) {
+
+    <div class="grid-row">
+        <div class="column-two-thirds">
+
+            <header class="page-heading">
+                <h1 class="heading-xlarge">
+                @messages("title.do_you_know_other_credentials.h1")
+                </h1>
+                <p class="heading-secondary"><span class="visuallyhidden">
+                    This section is </span>@messages("label.access_your_self_assessment")</p>
```

Comment:
Won't a screenreader go proper weird if this is in welsh? "This section is WELSH WORDS"